babel.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = function( api ) {
 		plugins: [
 			'babel-plugin-emotion',
 			'babel-plugin-inline-json-import',
+			'babel-plugin-dev-expression',
```

Comment:
Ideally, there would be an option in the plugin which ensures that a random function call with the caller name warning doesn’t get wrapped with the guard condition. In this case tracking the important statement would do the trick.

Reply:
I removed `babel-plugin-dev-expression` and included a `@wordpress/warning/babel-plugin` file in the package that checks the specifier name of any import declaration pointing to `@wordpress/warning`. It'll only transform function calls with that name.